api/v1/server/server.go

```diff
@@ -1,5 +1,7 @@
 // Code generated by go-swagger; DO NOT EDIT.
 
+// Hello
```

Comment:
👋

Reply:
Oops forgot to push.

Reply:
Doh, edited template but didn't regenerate.